presto-oracle/src/main/java/io/prestosql/plugin/oracle/OracleConfig.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.oracle;
+
+import io.airlift.configuration.Config;
+import io.airlift.units.Duration;
+import io.prestosql.plugin.jdbc.BaseJdbcConfig;
+
+import javax.validation.constraints.Min;
+
+import java.util.concurrent.TimeUnit;
+
+public class OracleConfig
+        extends BaseJdbcConfig
+{
+    private boolean includeSynonyms = true;
+    private boolean autoReconnect = true;
+    private int maxReconnects = 3;
```

Comment:
Remove `autoReconnect` and `maxReconnects`. As far as I know, those properties aren't supported in Oracle.

Reply:
you are correct..
chagned based on this: https://docs.oracle.com/cd/E11882_01/appdev.112/e13995/oracle/jdbc/OracleDriver.html

Reply:
We would like to add the minimum connector first, therefore some of the current configurations might be unnecessary. Let me check the details.